### PR TITLE
Rename Android TV to Android Debug Bridge

### PR DESCRIPTION
**Pull request by tronikos**
## Proposed change

Rename Android TV to Android debug bridge per Paulus' request in https://github.com/home-assistant/core/pull/89935#issuecomment-1493195164

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/90657
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/androidtv.markdown [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Android TV
-description: Instructions on how to integrate Android TV and Fire TV devices into Home Assistant.
+title: Android debug bridge
```
Comment (Drafteed) [inline source/_integrations/androidtv.markdown]: https://developer.android.com/studio/command-line/adb
```suggestion
title: Android Debug Bridge
```

File: source/_integrations/androidtv.markdown [hunk 2]
```diff
@@ -16,7 +16,7 @@ ha_platforms:
 ha_integration_type: device
 ---
 
-The `androidtv` platform allows you to control an Android TV device or [Amazon Fire TV](https://www.amazon.com/b/?node=8521791011) device.
+The `androidtv` integration allows you to control an Android device or [Amazon Fire TV](https://www.amazon.com/b/?node=8521791011) device.
```
Comment (Drafteed) [inline source/_integrations/androidtv.markdown]:
```suggestion
The Android Debug Bridge integration allows you to control an Android device or [Amazon Fire TV](https://www.amazon.com/b/?node=8521791011) device.
```

File: source/_integrations/androidtv.markdown [hunk 3]
```diff
@@ -211,25 +211,25 @@ to this:
...
+The Android debug bridge integration works by polling the Android / Fire TV device at a regular interval and collecting a handful of properties. Unfortunately, there is no standard API for determining the state of the device to which all apps adhere. Instead, the backend `androidtv` package uses three of the properties that it collects to determine the state: `audio_state`, `media_session_state`, and `wake_lock_size`. The correct logic for determining the state differs depending on the current app, and the backend `androidtv` package implements app-specific state detection logic for a handful of apps. Of course, it is not feasible to implement custom logic for each and every app in the `androidtv` package. Moreover, the correct state detection logic may differ across devices and device configurations.
```
Comment (Drafteed) [inline source/_integrations/androidtv.markdown]:
```suggestion
The Android Debug Bridge integration works by polling the Android / Fire TV device at a regular interval and collecting a handful of properties. Unfortunately, there is no standard API for determining the state of the device to which all apps adhere. Instead, the backend `androidtv` package uses three of the properties that it collects to determine the state: `audio_state`, `media_session_state`, and `wake_lock_size`. The correct logic for determining the state differs depending on the current app, and the backend `androidtv` package implements app-specific state detection logic for a handful of apps. Of course, it is not feasible to implement custom logic for each and every app in the `androidtv` package. Moreover, the correct state detection logic may differ across devices and device configurations.
```

**Comment by tronikos** [comment]
Thanks for your comments. Let's continue in https://github.com/home-assistant/home-assistant.io/pull/26832 since I think I had to use next instead of the current branch.

**Pull request closed without merging**